bio/example_test.go

```diff
@@ -0,0 +1,234 @@
+package bio_test
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/TimothyStiles/poly/bio"
+	"github.com/TimothyStiles/poly/bio/fasta"
+	"github.com/TimothyStiles/poly/bio/genbank"
+	"github.com/TimothyStiles/poly/bio/gff"
+	"github.com/TimothyStiles/poly/bio/polyjson"
+)
+
+// This is where the integration tests that make effed up cyclic dependencies go.
+func Example() {
+	// Poly can take in basic gff, gbk, fasta, and JSON.
+	// We call the json package "pson" (poly JSON) to prevent namespace collision with Go's standard json package.
+
+	gffInput, _ := gff.Read("../data/ecoli-mg1655-short.gff")
+	gbkInput, _ := genbank.Read("../data/puc19.gbk")
+	//fastaInput, _ := fasta.Read("fasta/data/base.fasta")
```

Comment:
why is fasta commented out?

Reply:
No `Read` functions anymore.

Reply:
Those could be added back in for each and every parser, it's just a bit annoying because how generic initialization works (ie, there would need to be a function for every type, like how parsing is done)